app/konnect/dev-portal/customization/custom.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Customize Kong Developer Portal
+no_version: true
+---
+
+The Kong Developer Portal currently offers three areas you can customize. 
+* [Appearance](#appearance)
+* [Custom URL](#add-a-custom-dev-portal-domain)
+* [Enabling Public Access](/konnect/dev-portal/publish/#access)
```

Comment:
I think this makes sense as a link list, since it's not necessarily functioning as a table of contents.